src/docs/timestamp-roundup.dox

```diff
@@ -15,34 +17,99 @@ Configuring <code>roundup_timestamps=(read=true)</code> causes the read
 timestamp to be rounded up to the oldest timestamp. If the read timestamp is
 greater than the oldest timestamp no change will be made.
 
-Configuring <code>roundup_timestamps=(prepared=true)</code> is only meaningful
-for transactions that will be prepared.
-It indicates that the prepare timestamp should be
-rounded up to the stable timestamp, if the prepare timestamp is less than the
-stable timestamp. This setting also indicates that the commit timestamp of the
-transaction should be rounded up to the prepare timestamp, if the commit
-timestamp is less than the prepare timestamp.
-It does not affect the durable timestamp.
-
-Based on the timestamp values and
-constraints, enabling this configuration could result in only one of the timestamps
-being rounded up. For example, for the timestamp values
-<code>prepare_timestamp=100, commit_timestamp=300, oldest_timestamp=200</code>
-with configuration <code>roundup_timestamps=(prepared=true)</code> only the
-prepare timestamp will be rounded up to the oldest timestamp. The commit
-timestamp will not be adjusted, and the result will be
-<code>prepare_timestamp=200, commit_timestamp=300, oldest_timestamp=200</code>.
-
-For cases where both the prepare timestamp and the commit timestamp need to be
-rounded up, first the prepare timestamp will be rounded to the oldest timestamp
-and then the commit timestamp will be rounded up to the new prepare timestamp.
-
-For example, for the timestamp values <code>prepare_timestamp=100,
-commit_timestamp=150, oldest_timestamp=200</code> with configuration
-<code>roundup_timestamps=(prepared=true)</code>, the prepare timestamp is
-rounded up to the oldest timestamp as <code>prepare_timestamp=200</code>
-when calling WT_SESSION::prepare_transaction.
-Subsequently as part of WT_SESSION::commit_transaction, the commit timestamp is
-rounded up to the new prepare timestamp as <code>commit_timestamp=200</code>.
+@section timestamp_roundup_prepare Replaying prepared transactions by rounding up the prepare timestamp
+
+It is possible for a system crash to cause a prepared transaction to
+be rolled back.
+Because the durable timestamp of a transaction is permitted to lag
```

Comment:
Is this sentence is telling that the durable timestamp can be lesser than the commit timestamp?

Reply:
It's not supposed to be, because that would be totally wrong :-)  By behind it's supposed to mean "after", but I agree that is totally not clear. Will fix (probably tomorrow).

Reply:
Fixed in 49e90cd, thanks.